README.md

```diff
@@ -174,6 +174,33 @@ quorumjs.extend(web3)
 
 Please see using Constellation and Quorum implementation private txn [example](https://github.com/jpmorganchase/quorum.js/blob/master/7nodes-test/deployContractViaIpc.js) and Tessera implementation [example](https://github.com/jpmorganchase/quorum.js/blob/master/7nodes-test/deployContractViaHttp.js). An extension sample is also provided.
 
+## Example for connecting to Tessera node with TLS enabled
+
+To send a request to Tessera node with TLS enabled for raw transactions, add the cert information as specified below. 
+
+```js
+
+let tlsOptions = {
+        key: fs.readFileSync('cert.key'),
+        clcert: fs.readFileSync('cert.pem'),
+        ca: fs.readFileSync('cert.pem')
+        allowInsecure: true
```

Comment:
I think it is better by default to start with `allowInsecure: false`. The next paragraph detailing the parameters is enough to suggest to use true for self signed certificates.

Reply:
@nicolae-leonte-go, sure, I'll update the readme.